openrtx/include/ui/EnglishStrings.h

```diff
@@ -32,6 +32,7 @@ const stringsTable_t englishStrings =
     .channels          = "Channels",
     .contacts          = "Contacts",
     .gps               = "GPS",
+    .spectrum          = "Spectrum",
```

Comment:
Please move this to the bottom of the list, to match the position to the one in the struct definition.
Also, add the string also to the spanish string table, otherwise the firmware is going to crash as soon as we implement multi-language support